demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>...we will use one-hot vectors that will be fed into a soft-max Keras layer during training.</blockquote><p> </p><p>These vectors are not input to the soft-max layer. They are used in calculating the loss by comparing with the output of the soft-max layer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>